### ISSUE TITLE
Could we get a push to pypi to support django-filter 2.x?

### ISSUE THREAD

**Issue by rrauenza** [philipn/django-rest-framework-filters#324]
Current version on pypi only supports django-filter 1.x -- your latest allows django-filter 2.x?

Is the project at a state where an update could be pushed?

Thanks!

**Comment by zarinpy** [comment]
I have the same problem ...

**Comment by intgr** [comment]
I had the same problem too. I found that it's possible to use the latest **pre-release** version `djangorestframework-filters==1.0.0.dev0`.

But I would like to know as well, how stable is it? Are there any known issues?

**Comment by mpistrang** [comment]
We also have the same question!

**Comment by caspervdw** [comment]
Same here

**Comment by joshorr** [comment]
Same!!!!

**Comment by codeadict** [comment]
Same here

**Comment by JoshYuJump** [comment]
```bash
raise VersionConflict(dist, req).with_context(dependent_req)
pkg_resources.ContextualVersionConflict: (django-filter 2.0.0 (/usr/local/python3.6/lib/python3.6/site-packages), Requirement.parse('django-filter~=1.1'), {'djangorestframework-filters'})
```
Same problem!

**Comment by tcarreira** [comment]
I guess nobody is watching this -.-

**Comment by LyleScott** [comment]
yeah, this is preventing some upgrades. it's been a part of a few upgrades we check up on every few months and it's been like this for a long time it seems. an update would be great.

**Comment by intgr** [comment]
Version 1.0.0.dev2 was pushed to PyPI in August 2020, so at least some development is happening :)

**Comment by mlissner** [comment]
There's a milestone. Looks like only documentation is holding it up: https://github.com/philipn/django-rest-framework-filters/issues?q=is%3Aopen+is%3Aissue+milestone%3Av1.0.0

**Comment by rrauenza** [comment]
> Version 1.0.0.dev2 was pushed to PyPI in August 2020, so at least some development is happening :)

I'm going to close this, then, since we have something we can work with now in pypi.

**Comment by mlissner** [comment]
Wait, I don't want to be a jerk, but relying on a dev version in pypi is a very bad solution.

Django filters 2.0 came out about a year and a half ago. We have code that works, but it's not properly released, making it difficult to upgrade to Django filters 2.0. Without first discovering this ticket. This problem is very much not solved.

Can we reopen this until we get a real release?

**Comment by ziXet** [comment]
Just wondering, what's exactly left for the first official release of 1.0.0?

**Comment by hwalinga** [comment]
@ziXet Only documentation: https://github.com/philipn/django-rest-framework-filters/issues?q=is%3Aopen+is%3Aissue+milestone%3Av1.0.0

**Comment by aqp696** [comment]
> @ziXet Only documentation: https://github.com/philipn/django-rest-framework-filters/issues?q=is%3Aopen+is%3Aissue+milestone%3Av1.0.0

You are right. But 3 open issues since 2016, 2018 and 2019 ... IMHO this project is abandoned or not properly maintained
